src/angular/core/datetime/native-date-adapter.ts

```diff
@@ -210,17 +219,35 @@ export class SbbNativeDateAdapter extends SbbDateAdapter<Date> {
     return !isNaN(date.valueOf());
   }
 
+  isOverflowingDate(value: string): boolean {
```

Comment:
Is it necessary for this method to be public? I would prefer it, if it was private or at least protected.